packages/create-miniapp-cli/src/helpers/format.ts

```
@@ -0,0 +1,24 @@
+import { type PackageManager } from '../utils/getUserPkgManager.js'
```

ESLint

[error] `../utils/getUserPkgManager.js` import should occur after import of `ora`

(import/order)